### DISCUSSION TITLE
Restore a deleted pub (with DOI)

### DISCUSSION THREAD

**Discussion by CASCS** [knowledgefutures/pubpub#2157] [Support] [answered]
Hi!

I deleted a pub, but I need to restore it. Is this possible?
If this is not possible, can I assign a specific DOI (deleted pub DOI) to a new pub?

Many thanks.

**Comment by gabestein** [comment, marked as answer]
Hi, and sorry for the delayed response.

We may be able to restore a Pub. Please email help@pubpub.org with the original link to it and the DOI assigned to it. If that's not possible, you can create a new one and we can set a specific DOI for it, yes.

**Reply by CASCS** [reply]
Thank you very much. I appreciate the time you took to answer my question.
